src/features/product/components/ProductTabs.tsx

```diff
@@ -36,16 +45,16 @@ export const ProductTabs = ({ productDetail, reviews }: ProductTabsProps) => {
       case 'reviews':
         return (
           <TabContent>
-            {reviews?.reviews && reviews.reviews.length > 0 ? (
+            {hasReviews ? (
               <ReviewsList>
-                {reviews.reviews.map((review, index: number) => (
+                {reviews!.reviews.map((review, index: number) => (
```

Comment:
실제 런타임에서 reviews가 null로 오게되면 죽을수도 있어서, non null assertion은 조심해서 사용하셔야해욥! 
아마 hasReviews로 위에서 조건부 했는데도 경고 떠서 그렇게 해주신 것 같아요 
아래처럼 사용하는 방법도 있답니다!

```suggestion
                {(reviews?.reviews ?? []).map((review, index: number) => (
```